README.md

```diff
@@ -92,9 +92,9 @@ To generate reports, execute CloudAI CLI in generate-report mode:
 cloudai generate-report\
     --system-config conf/common/system/example_slurm_cluster.toml\
     --tests-dir conf/common/test\
-    --output-dir /path/to/output_directory
+    --results-dir /path/to/results_directory
 ```
-In the generate-report mode, use the --output-dir argument to specify a subdirectory under the result directory.
+In the generate-report mode, use the --results-dir argument to specify a subdirectory under the result directory.
```

Comment:
Sorry to nitpick ... this should be "In the generate-report mode, use the --results-dir argument to specify a subdirectory under the output directory."... correct?

Reply:
It makes sense considering that it is part of the output directory. Updated.